### PR TITLE
almost all answers to PR6

### PR DESCRIPTION
**Pull request by kathypinzon**
answered all but the last two - I need Abby to check those questions because I think she wants the denominator to be +6 instead of -6

### CODE REVIEW

**Comment by kathypinzon** [comment]
@AbbyANoble can you check [Activity 4.6.19](https://b2ac9052.precalculus-beta-20240616.pages.dev/2024/instructor/PR6#activity-rf-holes-extension) 
I think you want +6 not -6 in denominator.

**Review by siwelwerd — APPROVED** [review]

**Comment by siwelwerd** [comment]
Approved, but holding off on merging pending the resolution of the question above.

**Comment by AbbyANoble** [comment]
This was one I grabbed from Active Prelude and now that I look closer it looks like it doesn't quite do what I thought it did. I'll make some edits and do a pull request tomorrow.

**Pull request merged**
